src/api/app/controllers/source_project_meta_controller.rb

```diff
@@ -70,10 +70,12 @@ def update
       else
         project = Project.new(name: @project_name)
         project.update_from_xml!(@request_data)
-        # FIXME3.0: don't modify send data
-        maintainer_role = Role.find_by_title!('maintainer')
-        unless project.relationships.any? { |relationship| relationship.user == User.session! && relationship.role == maintainer_role }
-          project.relationships.find_or_initialize_by(user: User.session!, role: maintainer_role)
+        unless CONFIG['prevent_adding_maintainer_in_project_creation_with_api']
```

Comment:
People tend to expect that those options are on/off switches. So they will change your example to `= true` and wonder why it isn't working. `Configuration` has some support for this, check it out.